### PR TITLE
feat: Modals: Maximum form cloning & Input image validation

### PR DESCRIPTION
**Pull request by hatdogguldo**
1. I added a program key for candidate table to store different organization program or majors: [Link](https://drive.google.com/drive/folders/1tKJrlJ9c2YdfNwNNtgLMUgcDOk-FRHtz?fbclid=IwAR2OB3lpLjHFirKu7OI_LAEu1Jt53dfxtAdG2032GfDB0lROGn8hSms1uiw)

note: The cloned form's input validation is currently under development. Just fixing the said issue to close this pr, thank you.

### CODE REVIEW

**Comment by Carl-Tabuso** [comment]
Assigning the review for this pull request to @biellamariscotes

File: src/add-candidate.php [hunk 1]
```diff
@@ -16,65 +16,65 @@
...
                             </div>
                         </div>
                     </div>
+                </div>
 
                 <form action="../src/submission_handlers/insert-candidates.php" method="post" id="candidate-form" enctype="multipart/form-data">
                     <div class="container">
                         <div class="row justify-content-center">
-                            <div class="col-md-10 card-box mt-md-10" id = "carl">
+                            <div class="col-md-10 card-box mt-md-10" id="carl">
```
Comment (biellamariscotes) [inline src/add-candidate.php]: "carl"? Please provide a clearer, more descriptive name for the ID.

File: src/add-candidate.php [hunk 2]
```diff
@@ -84,156 +84,245 @@
...
+            <!-- Created Maximum Modal -->
+            <div class="modal fade" id="errorModal" data-bs-keyboard="false" data-bs-backdrop="static">
+                <div class="modal-dialog modal-dialog-centered" role="document">
+                    <div class="modal-content">
+                        <div class="modal-body">
+                            <div class="text-center">
+                                <div class="modal-header">
                                     <div class="col-md-12">
-                                        <img src="images/resc/check-animation.gif" class="check-perc" alt="iVote Logo">
+                                        <img src="images/resc/warning.png" alt="Warning Icon">
```
Comment (biellamariscotes) [inline src/add-candidate.php]: Incorrect vector, do replace with [yellow-warning.png](https://drive.google.com/drive/folders/1H1Z7z9CHJZF_-jIwka9CoWR-DJYe34Cx?usp=sharing).

File: src/add-candidate.php [hunk 3]
```diff
@@ -84,156 +84,245 @@
...
                                     </div>
-                                    <div class="row">
-                                        <div class="col-md-12 pb-3">
-                                            <p class="fw-bold fs-3 success-color spacing-4">Successfully Created!</p>
-                                        </div>
+                                </div>
+
+                                <div class="row">
+                                    <div class="col-md-12 pb-3 pt-4">
+                                        <p class="fw-bold fs-6 text-warning spacing-4 px-2">Maximum limit. You can only add up to 5 candidates at a time</p>
```
Comment (biellamariscotes) [inline src/add-candidate.php]: Have a title and a sub-title for this:

```
<p class="fw-bold fs-3 text-warning spacing-4">Maximum limit reached.</p>
<p class="fw-medium spacing-5">You can only add up to 5 candidates at a time.</p>
```

**Review by biellamariscotes — CHANGES_REQUESTED** [review]
See comments in the codes.

**Comment by biellamariscotes** [comment]
In **Add Candidate**, attempting to add more than 2 results in failure and errors in error.log:


https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/135019439/11a73d55-9220-48dd-adae-31ca3ce60851

```
2024-06-15 10:28 pm Error: [8] session_start(): Ignoring session_start() because a session is already active - C:\xampp\htdocs\PUPSRC-AutomatedElectionSystem\src\submission_handlers\insert-candidates.php: 7
2024-06-15 10:28 pm Error: [2] Undefined array key "party_list" - C:\xampp\htdocs\PUPSRC-AutomatedElectionSystem\src\submission_handlers\insert-candidates.php: 19
2024-06-15 10:28 pm Error: [8192] trim(): Passing null to parameter #1 ($string) of type string is deprecated - C:\xampp\htdocs\PUPSRC-AutomatedElectionSystem\src\submission_handlers\insert-candidates.php: 19
2024-06-15 10:28 pm Error: [2] Undefined array key "election_year" - C:\xampp\htdocs\PUPSRC-AutomatedElectionSystem\src\submission_handlers\insert-candidates.php: 21
2024-06-15 10:28 pm Error: [8192] trim(): Passing null to parameter #1 ($string) of type string is deprecated - C:\xampp\htdocs\PUPSRC-AutomatedElectionSystem\src\submission_handlers\insert-candidates.php: 21
```

It only submits the first form, and fails to successfully query the subsequent forms.

<hr>

Also, for the **X** mark on your forms, please re-style it to this:

image

File: src/add-candidate.php (hunk 3 above)
Comment (hatdogguldo) [reply src/add-candidate.php]: Noted

File: src/add-candidate.php (hunk 2 above)
Comment (hatdogguldo) [reply src/add-candidate.php]: Noted

File: src/add-candidate.php (hunk 1 above)
Comment (hatdogguldo) [reply src/add-candidate.php]: Noted

**Comment by biellamariscotes** [comment]
Following up on this PR. Please address the requested changes as soon as possible. @hatdogguldo

**Comment by hatdogguldo** [comment]
@biellamariscotes kindly disregard this [commit](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/commit/14fe1386083a0e99a3ea8d2532b5ebf0a8bbbca1). This should considered as new PR, not a commit for this pr.

**Review by biellamariscotes — APPROVED** [review]

**Pull request merged**